src/test/fuzz/deserialize.cpp

```diff
@@ -189,7 +193,9 @@ FUZZ_TARGET_DESERIALIZE(blockmerkleroot, {
     BlockMerkleRoot(block, &mutated);
 })
 FUZZ_TARGET_DESERIALIZE(addrman_deserialize, {
-    AddrMan am(/* asmap */ std::vector<bool>(), /* deterministic */ false, /* consistency_check_ratio */ 0);
+    AddrMan am(/* asmap */ std::vector<bool>(),
+               /* deterministic */ false,
+               gArgs.GetIntArg("-checkaddrman", 0));
```

Comment:
It might be clearer to use `testing_setup.m_node.args`, since this is where they were put.

Reply:
Done, it was necessary to define the `testing_setup` variable as a global, outside of the `initialize_...()` function so that it can be accessed by this code.